lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -171,50 +170,15 @@ void _showRemoveGuardianDialog(BuildContext buildContext, GuardianModel guardian
   showDialog(
     context: buildContext,
     builder: (BuildContext context) {
-      return AlertDialog(
-        content: Column(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            const Text(
-              "Are you sure you want to remove ",
-            ),
-            const SizedBox(height: 16),
-            ListTile(
-              leading: ProfileAvatar(
-                size: 60,
-                image: guardian.image,
-                account: guardian.uid,
-                nickname: guardian.nickname,
-                decoration: const BoxDecoration(
-                  shape: BoxShape.circle,
-                  color: AppColors.blue,
-                ),
-              ),
-              title: Text(
-                "${guardian.nickname}",
-                style: const TextStyle(color: Colors.black),
-              ),
-              subtitle: Text("${guardian.uid}"),
-            ),
-            const SizedBox(height: 16),
-            const Text("As your Guardian?"),
-          ],
-        ),
-        actions: [
-          TextButton(
-            child: const Text('Dismiss'),
-            onPressed: () {
-              Navigator.pop(context);
-            },
-          ),
-          TextButton(
-            child: const Text('Remove Guardian'),
-            onPressed: () async {
-              BlocProvider.of<GuardiansBloc>(buildContext).add(OnRemoveGuardianTapped(guardian));
-              Navigator.pop(context);
-            },
-          )
-        ],
+      return RemoveGuardianConfirmationDialog(
+        guardian: guardian,
+        onConfirm: () {
+          BlocProvider.of<GuardiansBloc>(buildContext).add(OnRemoveGuardianTapped(guardian));
+          Navigator.pop(context);
+        },
+        onDismiss: () {
+          Navigator.pop(context);
+        },
```

Comment:
Am I the only one who likes to use arrow functions when it is only a line?

1 line instead 3 sounds like a good visual deal ;)